MANIFEST.in

```diff
@@ -0,0 +1,10 @@
++include README.md
++include prompt.py
++
```

Comment:
doesn't it need to include `neo` and all subdirectories too here?

Reply:
Thanks, you are right. Just fixed this.